docs(data): improve DefaultDataServiceConfig root parameter docs (#2075)
ngrx/platform: projects/ngrx.io/content/guide/data/entity-dataservice.md

The NgRx Data library defines a `DefaultDataServiceConfig` for 
conveying shared configuration to an entity collection data service.

> The most important configuration property, `root`, returns the _root_ of every web api URL, the parts that come before the entity resource name. If you are using a remote API, this value can include the protocol, domain, port, and root path, e.g.: `'https://my-api-domain.com:8000/api/v1'`.

Reviewer: ```suggestion
The most important configuration property, `root`, returns the _root_ of every web api URL, the parts that come before the entity resource name. If you are using a remote API, this value can include the protocol, domain, port, and root path, such as `https://my-api-domain.com:8000/api/v1`.
```
